src/contracts/deposits-registry/store/store.service.ts

```diff
@@ -165,6 +192,17 @@ export class LevelDBService {
     await this.db.batch(ops);
   }
 
+  /**
+   * Inserts a batch of deposit events and a header into the database.
+   *
+   * @param {VerifiedDepositEvent} event - Last valid and verified event.
+   * @returns {Promise<void>} A promise that resolves when all operations have been successfully committed to the database.
+   * @public
+   */
+  public async insertLastValidEvent(event: VerifiedDepositEvent) {
+    await this.db.put('last-valid-event', this.serializeDepositEvent(event));
```

Comment:
TODO: 'last-valid-event' in LAST_VALID_EVENT

Reply:
I'm not sure I should write it that way, in all the other places we've written it in lower case.

Reply:
i mean place in a constant